profiler/options.go

```diff
@@ -71,6 +72,29 @@ func urlForSite(site string) (string, error) {
 	return u, err
 }
 
+// isAPIKeyValid reports whether the given string is a structurally valid API key
+func isAPIKeyValid(key string) bool {
+	if len(key) != 32 {
+		return false
+	}
+	for _, c := range key {
+		if !unicode.IsLetter(c) && !unicode.IsLower(c) && !unicode.IsNumber(c) {
+			return false
+		}
+	}
+	return true
+}
+
+// sanitizeAPIKey prepares an API key for display, by censoring everything
+// beyond the first 8 characters of the input
+func sanitizeAPIKey(key string) string {
```

Comment:
I think you're right making a point in your other thread: we shouldn't print this out. So we can remove this and just print the message. What do you think?

Reply:
Sounds good to me, will remove.